src/Main.hs

```diff
@@ -218,9 +220,10 @@ renderPage server baseUrl page = with html_ [lang_ "en"] $ do
             _topicLastUpdated t
           whenJust (listToMaybe $ _topicMessages t) $ \msg -> do
             ogpAttribute "article:published_time" $ ogpTimeFormat $ _messageTimestamp msg
-            -- FIXME: _messageContent is HTML, but we need plain text.
-            ogpAttribute "description" $ T.take 300 $ _messageContent msg
+            ogpAttribute "description" $ T.take 300 $ stripHtml $ _messageContent msg
             ogpAttribute "image" `mapM_` _messageAvatarUrl msg
+    stripHtml :: T.Text -> T.Text
```

Comment:
`T.Text` should be replaced with `Text` (it is available from `relude` which the app uses as its prelude).